apps/data-pipeline/ap-exams-importer/src/data.ts

```diff
@@ -0,0 +1,737 @@
+    acceptableScores: (1 | 2 | 3 | 4 | 5)[];
+    // units granted, most relevant for residency requirement
+    unitsGranted: number;
+    // units of catch-all elective credit granted not through a course
+    electiveUnitsGranted: number;
+    // GEs granted directly and not through a course
+    geGranted: (typeof geCategories)[number][];
+    coursesGranted: APCoursesGrantedTree;
+  }[];
+};
+type Mapping = Record<string, APExam>;
+
+export default {
+  "AP African American Studies": {
+    // Satisfies one year of U.S. History for the Area A requirement.
+    rewards: [
+      {
+        acceptableScores: [3, 4, 5],
+        unitsGranted: 4,
+        electiveUnitsGranted: 0,
+        geGranted: [],
+        coursesGranted: { AND: [] },
+      },
+    ],
+  },
+  "AP Art History": {
+    rewards: [
+      // One course toward Art History major, minor, category IV of the UCI GE requirement as ART HIS 40A, and
+      // satisfaction of category VIII, plus 4 units of elective credit; may not replace School of Humanities
+      // requirements
+      {
+        acceptableScores: [3],
+        unitsGranted: 8,
+        electiveUnitsGranted: 4,
+        geGranted: [],
+        coursesGranted: { AND: ["ART HIS 40A"] },
+      },
+      // Two courses toward Art History major, minor, category IV of the UCI GE requirement as ART HIS 40A and
+      // ART HIS 40B, and satisfaction of category VIII; may not replace School of Humanities requirements
+      {
+        acceptableScores: [4, 5],
+        unitsGranted: 8,
+        electiveUnitsGranted: 4,
+        geGranted: [],
+        coursesGranted: { AND: ["ART HIS 40A", "ART HIS 40B"] },
+      },
+    ],
+  },
+  "AP Drawing": {
+    rewards: [
+      // Elective credit only
+      {
+        acceptableScores: [3, 4, 5],
+        unitsGranted: 8,
+        electiveUnitsGranted: 8,
+        geGranted: [],
+        coursesGranted: { AND: [] },
+      },
+    ],
+  },
+  "AP 2-D Art and Design": {
+    rewards: [
+      // Elective credit only
+      {
+        acceptableScores: [3, 4, 5],
+        unitsGranted: 8,
+        electiveUnitsGranted: 8,
+        geGranted: [],
+        coursesGranted: { AND: [] },
+      },
+    ],
+  },
+  "AP 3-D Art and Design": {
+    rewards: [
+      // Elective credit only
+      {
+        acceptableScores: [3, 4, 5],
+        unitsGranted: 8,
+        electiveUnitsGranted: 8,
+        geGranted: [],
+        coursesGranted: { AND: [] },
+      },
+    ],
+  },
+  "AP Biology": {
+    rewards: [
+      // Non-Bio. Sci. majors earn one Biological Sciences course toward Category II of the UCI GE requirement.
+      // Bio. Sci. majors earn elective credit only
+      {
+        acceptableScores: [3, 4, 5],
+        unitsGranted: 8,
+        electiveUnitsGranted: 8,
+        geGranted: [],
+        coursesGranted: { AND: [] },
+      },
+    ],
+  },
+  // Capstone Research and Seminar: "Approved as Area G college-prep elective only. No university credit awarded"
```

Comment:
missing AP capstone? or I guess just informative comment that happens to be next to AP chem

Reply:
capstone is the combination of research and seminar

it is indeed an informative comment to indicate that this row does exist in UCI's table but is not in our data for the reason given